frontend/src/app/SiteUtils/SiteUtils.tsx

```diff
@@ -36,11 +39,48 @@ const SiteUtils: React.FunctionComponent = () => {
   ]
 
   const [message, setMessage] = useState('');
-  const [mediaType, setMediaType] = useState('');
   const [severity, setSeverity] = useState('');
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [feedbackMessage, setFeedbackMessage] = useState('Succeed');
+
+  async function onUpdateBanner() {
+    setFeedbackMessage('Succeed');
+    if (message && severity) {
+       axios.put('/update_banner', {
+        message,
+        severity
+      })
+      .then(function (response) {
+        setIsModalOpen(true);
+      })
+      .catch(function (error) {
+        setFeedbackMessage(error.response.data);
+        setIsModalOpen(true);
+      });
+    }
+  }
+
+  async function onClearBanner() {
+    setFeedbackMessage('Succeed');
```

Comment:
the `Succeeded` message should be sent after the request has returned successfully